include/usb/class/usb_hid.h

```diff
@@ -69,6 +70,7 @@ struct hid_ops {
 	 * the next transfer.
 	 */
 	hid_int_ready_callback int_in_ready;
+	hid_out_ready_callback int_out_read;
```

Comment:
Wouldn't it be better to enable an INT OUT endpoint based on a configuration variable? Since Endpoints may be a premium in devices, whether or not an INT OUT is added to the system should be a choice and a configuration variable would make it easy to make this choice.